services/brig/test/integration/API/User/Util.hs

```diff
@@ -245,6 +247,30 @@ listConnections brig u =
       . path "connections"
       . zUser u
 
+listAllConnections :: (MonadIO m, MonadHttp m) => Brig -> UserId -> Maybe Int -> Maybe (MultiTablePagingState "Connections" LocalOrRemoteTable) -> m ResponseLBS
```

Comment:
You have `expect2xx` in there so I'd say it would be useful to have `HasCallStack` in the constraint list.